### PR TITLE
RTEMIS-127: Created a custom module for demo content.

### PR DESCRIPTION
**Pull request by RohanSingh-inno**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-127

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Created a custom module for demo content.

### CODE REVIEW

File: modules/rte_mis_demo_content/src/TestData.php [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+    // Check for existing active campaign before creating one.
+    $existing_active_campaign = $storage->loadByProperties([
+      'type' => 'academic_session',
+      'field_academic_year' => '2024_25',
+      'status' => 1,
+    ]);
+    if (empty($existing_active_campaign)) {
+      $node = $storage->create([
+        "type" => "academic_session",
+        "field_academic_year" => '2024_25',
```
Comment (jaswinsingh) [inline modules/rte_mis_demo_content/src/TestData.php:297]: use single quotes

File: modules/rte_mis_demo_content/src/TestData.php [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+      ]);
+      $paragraphs = $this->getTimelineParagraphs();
+      $node->set("field_session_details", $paragraphs)->save();
```
Comment (jaswinsingh) [inline modules/rte_mis_demo_content/src/TestData.php:300]: Same here, use single quotes

Comment (RohanSingh-inno) [reply modules/rte_mis_demo_content/src/TestData.php:300]: ok

File: modules/rte_mis_demo_content/src/TestData.php (hunk 1 above)
Comment (RohanSingh-inno) [reply modules/rte_mis_demo_content/src/TestData.php:297]: ok

File: modules/rte_mis_demo_content/src/TestData.php [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+    $storage = $this->entityTypeManager->getStorage('mini_node');
+    // Check for existing active campaign before creating one.
+    $existing_active_campaign = $storage->loadByProperties([
+      'type' => 'academic_session',
+      'field_academic_year' => '2024_25',
+      'status' => 1,
+    ]);
+    if (empty($existing_active_campaign)) {
+      $node = $storage->create([
+        'type' => 'academic_session"',
```
Comment (jaswinsingh) [inline modules/rte_mis_demo_content/src/TestData.php:296]: Please check here, this will lead to fatal error.

Comment (RohanSingh-inno) [reply modules/rte_mis_demo_content/src/TestData.php]: fixed.

**Review by jaswinsingh — APPROVED** [review]

**Pull request merged**